freezeyt/cli.py

```diff
@@ -24,9 +24,13 @@
 @click.option('--progress', 'progress',
               type=click.Choice(['none', 'bar', 'log']),
               help='Select how to display progress')
+@click.option('-rid', '--remove-incomplete-dir', 'remove_incomplete_dir',
```

Comment:
Short options (with one dash) should only have one letter. You can also only have the long option.

For yes/no/default options, see: https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags
Generally: for a boolean value you always want to deal with True/False in code. If you use something else (like `"yes"`/`"no"`) for input from the user, convert to bool as early as possible.

Also note that YAML converts `yes` and `no` (without quotes) to booleans.